app/models/miq_expression.rb

```diff
@@ -1496,7 +1496,12 @@ def self.build_relats(model, parent = {}, seen = [])
     result = {:columns => model.column_names_with_virtual, :parent => parent}
     result[:reflections] = {}
 
-    model.reflections_with_virtual.each do |assoc, ref|
+    refs = model.reflections_with_virtual
+    model.descendants.each do |desc|
```

Comment:
Yeah, I was concerned about that too but couldn't come up with a better way to dynamically pull in the descendant class reflections without loading everything in. I considered only loading the ones they needed but that would force us to continually update it.